src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -62,7 +73,30 @@ class EntityClient (requestContext: RequestContext) extends Actor {
         requestContext.failWith(
           new RequestProcessingException(StatusCodes.InternalServerError, error.getMessage))
     }
+  }
 
+  def createEntities(wsNamespace: String, wsName: String, entities: Seq[Entity]) = {
+    val results = entities.map(entity => createEntityOrReportError(wsNamespace, wsName, entity))
+    // have to manually construct an HttpResponse here because we're aggregating the results of multiple calls to Rawls
+    val response = HttpResponse(OK, HttpClient.createJsonHttpEntity(results.toJson.compactPrint))
```

Comment:
I think you need an intermediary service handler to track entity handling at the individual level. If all succeed, then definitely `Created` should be the return, potentially with a list of all of the created entities or ids, whatever makes the most sense.

For the negative case, track any failures and possibly return an error with the reasons for the failures. I think a 422, "Unprocessable Entity" might be the most logical response here. The service handler should likely call delete on any of the successfully created entities so as to clean up after itself.